src/Raven.Server/Routing/RouteInformation.cs

```diff
@@ -81,6 +81,8 @@ public void Build(MethodInfo action)
         public Task CreateDatabase(RequestHandlerContext context)
         {
             var databaseName = context.RouteMatch.GetCapture();
+
+            // todo: think if we need to pass this check to the landlord
```

Comment:
@maximburyak is that TODO still relevant?

Reply:
Yes, not something that in the scope of this PR, but it's a nice thing to do.

Reply:
Can you create issue for that?

Reply:
NP: https://issues.hibernatingrhinos.com/issue/RavenDB-14307